content/hardware/05.pro-solutions/solutions-and-kits/wisgate-edge-lite-2/tutorials/update-firmware/content.md

```diff
@@ -57,7 +57,7 @@ On the WisGate Dashboard, open the Settings page and click on the "Firmware" tab
 
 ![WisGate dashboard Settings page](assets/wisgate-firmware-settings.png)
 
-Now drag and drop or open the browse file option to select the firmware file downloaded in the previous step, it is the file with the `.bin.rwi` format, as you can see in the screenshot we uploaded the file called `WisGateOS_2.0.1_ARDUINO_b30_RAK636.bin.rwi`
+Now drag and drop or open the browse file option to select the firmware file downloaded in the previous step, it is the file with the `.bin.rwi` format, as you can see in the screenshot we uploaded the file called `WisGateOS_2.1.4_ARDUINO_b87_RAK636.bin.rwi`
```

Comment:
@Jorgetrujilloroman did we uploaded a new screenshot?

Change this to be software version agnostic

This way we can maintain better the documentation

Instead of saying the name of the version say something like:

```suggestion
Now drag and drop or open the browse file option to select the firmware file downloaded in the previous step, it is the file with the `.bin.rwi` format, as you can see in the screenshot.
```

This way we can maintain the same screenshot and we don't need to change the text everytime we have a new version

Reply:
Great suggestion, commiting changes